packages/manager/apps/pci-instances/src/pages/instances/create/CreateInstance.page.tsx

```diff
@@ -39,8 +39,8 @@ export const instanceCreationSchema = z.object({
   availabilityZone: availabilityZoneSelectionSchema,
   distributionImageType: distributionImageTypeSchema,
   distributionImageName: distributionImageNameSchema,
-  sshName: sshNameSchema.nullable(),
-  sshKey: sshKeySchema.optional(),
+  sshKeyId: sshKeyIdSchema,
+  newSshPublicKey: sshPublicKeySchema.nullable(),
```

Comment:
could you integrate nullable into `sshPublicKeySchema`? as you did for `sshKeyIdSchema`